### DISCUSSION TITLE
有没有大佬带带我呀？

### DISCUSSION THREAD

**Discussion by fu-fu404** [share121/inter-knot#964] [其他]
本人绝区零小萌新，有意带我请回帖

**Comment by BITE-7274** [comment]
现在不是没有联机吗
怎么带
贪吃蛇吗·-·

**Reply by fu-fu404** [reply]
我就是打怪打不过，圣遗物什么的没接触过，根本不会配

**Comment by ZMJ2333** [comment]
诶嘿，这个的话可以多多光顾一下[米游社](https://baike.mihoyo.com/zzz/wiki/?mhy_presentation_style=fullscreen&visit_device=pc&utm_source=bbs&utm_medium=zzz&utm_campaign=icon)
> [!TIP]
> 部分代理人可以按照ta们的 **元素** 来匹配驱动盘，看四件套加的什么就行
> 还有一些是看 **职业** 比如强攻要加攻击、双爆，异常要加异常精通，击破要加冲击力等等
> 最后是最简单的：角色面板有一些数值被 *标黄* 了，那些数值就是应该被着重提升的
